addons/mail/static/src/core/common/quick_reaction_menu.js

```diff
@@ -102,9 +102,14 @@ export class QuickReactionMenu extends Component {
     }
 
     get attClass() {
-        const invisible = !this.props.messageActive && !this.dropdown.isOpen && !this.picker.isOpen;
+        const invisible =
+            typeof this.props.messageActive === "boolean" &&
+            !this.props.messageActive &&
```

Comment:
`messageActive` is optional prop, this was enforcing `invisible` because this is too much coupling with `Message` component with visibility of the quick actions.